bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/api/BStringValues.java

```diff
@@ -41,7 +53,9 @@
  * 
  * @since 0.95.3
  */
-public class StringUtils {
+public class BStringValues {
```

Comment:
should this be name BStringCreator ?

Reply:
It has Bstring to java string and vice versa apis. So "BstringCreator" not much suited for the class. Any other suggestions?

Reply:
the shall we use BStringUtils ?

Reply:
+1